frontend/online-shop/src/components/ProductDetails.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  Button,
+  Card,
+  CardContent,
+  CardHeader,
+  CardMedia,
+  Grid,
+  Typography,
+} from "@material-ui/core";
+import { Link, useParams } from "react-router-dom";
+import ShoppingCartIcon from "@material-ui/icons/ShoppingCart";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  customGrid: {
+    justifyContent: "center",
+  },
+  customText: {
+    color: theme.palette.primary.main,
+  },
+  customCard: {
+    borderStyle: "none",
+    borderRadius: 3,
+  },
+}));
+
+function ProductDetails(props: any) {
+  // const params = useParams()
```

Comment:
I saw your comment saying why you left this here. Usually we don't do this unless it's some really important/hard to write piece of code.